test/addons/velero/start

```diff
@@ -24,11 +25,30 @@ def deploy(cluster):
         "--use-volume-snapshots=false",
         f"--backup-location-config=region=minio,s3ForcePathStyle=true,s3Url={s3_url}",
         f"--kubecontext={cluster}",
-        "--wait",
```

Comment:
We don't have an option to control the wait timeout? This change seems to duplicate velero tool functionality.

If we don't find a way to configure the wait time, please open an issue in velero github about this.